frontend/src/environments/environment.ssr.ts

```diff
@@ -2,6 +2,7 @@ export const environment = {
   apiRoot: 'http://backend:8000/api',
   djangoRoot: 'http://backend:8000/o',
   production: false,
+  randomOfferImages: true,
```

Comment:
I'm not really up to date with the project, but do we want to generate random images on the server side? Won't it cause i.e. sharing on facebook to display this kind of random images?

Or this is just the "dev" ssr?

Reply:
I believe, during SSR this image will be once randomly chosen and browser will do it again. As I wrote in my comment below I will work on choosing random images inside factories.

That way we will have random logic moved to factories and avoid having it in frontend at all.